todo_app/app.py

```diff
@@ -1,15 +1,22 @@
-from flask import Flask
+from flask import Flask, render_template, redirect, url_for, request
 
 from todo_app.flask_config import Config
+from todo_app.data.session_items import get_items, add_item
 
 app = Flask(__name__)
 app.config.from_object(Config)
 
 
-@app.route('/')
+@app.route('/', methods=['GET'])
 def index():
-    return 'Hello World!'
-
+    items3 = get_items()
```

Comment:
Oops, we forgot to undo this after our name change! As we did elsewhere, we could change this to something more helpful (like `todos`) if that is clearer